spec/amp-html-layout.md

```diff
@@ -83,18 +81,55 @@ AMP provides a set of layouts that specify how an AMP component behaves in the d
+          <li> if <code>width</code> and <code>height</code> are absent, a <code>container</code> layout is assumed.</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>container</code></td>
+      <td>The element lets its children define its size, much like a normal HTML <code>div</code>. The component is assumed to not have specific layout itself but only act as a container; its children are rendered immediately.</td>
+    </tr>
+    <tr>
+      <td><code>fill</code></td>
+      <td>The element takes the space available to it&mdash;both width and height. In other words, the layout and size of a <code>fill</code> element matches its parent.</td>
+    </tr>
+    <tr>
+      <td><code>fixed</code></td>
+      <td>The element has a fixed width and height with no responsiveness supported. The <code>width</code> and <code>height</code> attributes must be present. The only exceptions are the <code>amp-pixel</code> and <code>amp-audio</code> components. </td>
+    </tr>
+    <tr>
+      <td><code>fixed-height</code></td>
+      <td>The element takes the space available to it but keeps the height unchanged. This layout works well for elements such as <code>amp-carousel</code> that involves content positioned horizontally. The <code>height</code> attribute must be present. The <code>width</code> attribute must not be present or must be equal to <code>auto</code>. </td>
+    </tr>
+    <tr>
+      <td><code>flex-item</code></td>
+      <td>The element and other elements in its parent with layout type <code>flex-item</code> take the parent container's remaining space when the parent is a flexible container (i.e., <code>display: flex</code>). The <code>width</code> and <code>height</code> attributes are not required.</td>
+    </tr>
+    <tr>
+      <td><code>nodisplay</code></td>
+      <td>The element isn't displayed, and takes up zero space on the screen as if its display style was <code>none</code>. This layout can be applied to every AMP element.  It’s assumed that the element can display itself on user action (e.g., <code>amp-lightbox</code>). The <code>width</code> and <code>height</code> attributes are not required.</td>
+    </tr>
+    <tr>
+      <td><code>responsive</code></td>
+      <td>The element takes the space available to it and resizes its height automatically to the aspect ratio given by the <code>width</code> and <code>height</code> attributes. This layout works very well for most AMP elements, including <code>amp-img</code>, <code>amp-video</code>, etc.  The available space depends on the parent element and can also be customized using <code>max-width</code> CSS. The <code>width</code> and <code>height</code> attributes must be present.<br><em>Note</em>: Fixed positioned elements do not have an inherent width. Therefore, if the parent of a responsive element is a fixed position element, the responsive element will also have zero width.</td>
```

Comment:
I'm not sure that documenting this special is important enough since it will be only valuable to few but confuse many.

In any case lets make it:

Fixed positioned elements do not have an inherent width. Therefore you must give it width (such as with `width: 100%)` in order to use responsive elements inside of it.